nodejs/exceljs/getConnectionString.js

```diff
@@ -0,0 +1,24 @@
+require('dotenv').config();
+
+function getConnectionString() {
+    let connectionString = 'DSN=*LOCAL;';
+
+    if (process.env.DB_USER) { // using uid/password to connect
+      connectionString = `DRIVER=IBM i Access ODBC Driver;UID=${process.env.DB_USER};`;
+   
+      if (!process.env.DB_PASS) {
+        throw Error('"DB_PASS" environment variable must be defined');
+      }
+
+      connectionString += `PWD=${process.env.DB_PASS};`;
+      connectionString += process.env.DB_HOST ? `SYSTEM=${process.env.DB_HOST};`
+                                              : `SYSTEM=${localhost};`;
+    } else if (process.env.DB_DSN) { // using DSN to connect
```

Comment:
In the false end of the ternary should default to 'localhost' string. There is no localhost variable.